RationalNumber.cs

```diff
@@ -0,0 +1,115 @@
+﻿namespace Task4._2
+{
+	internal class RationalNumber :IComparable<RationalNumber>
+	{
+        public int Numerator { get; private set; }
+		private int denominator;
+
+		public int Denominator
+		{
+			get { return denominator; }
+			private set {
+				if (value == 0)
+				{
+					throw new ArgumentException("Denominator must be different from 0!");
+				}	
+				denominator = value; 
+			}
+		}
+
+
+		private RationalNumber(int n, int m)
+        {
+            this.Numerator = n;
+            this.Denominator = m;
+			Rationalize();
+        }
```

Comment:
Code style (for method name)
Why do you implement this method instead of public constuctor?